index.html

```diff
@@ -0,0 +1,348 @@
+
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-web">Website Protfolio </h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class=" desk-prof-art">Profesional Art Printing Data</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-web">Website Protfolio </h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+
+  </section>
+
+  <section class="desk-cards">
+    <div class="desk-h21-line">
+      <h1 class="desk-h21">My Recent Works</h1>
+      <hr class="desk-line">
+    </div>
```

Comment:
- [ ] As explained by the previous reviewer the spaces on the sides of all the sections are not matching the Figma design as shown below 🔽, please try to fix it

![spaces](https://user-images.githubusercontent.com/104319462/221053584-691e5e1d-bce7-4c0b-b507-473dd8d2f460.png)